string image_path
---
uint8 success
string message
